src/partials/products.html

```diff
@@ -0,0 +1,38 @@
+<section class="products">
+  <div class="products-head">
+    <p class="products-head__text">100&percnt; natural</p>
+    <h1 class="products-head__title" id="products">products</h1>
+  </div>
+  <ul class="products-grid">
+    <li class="product-box product-box--bg-color">
+      <p class="product-box__title">ice cream</p>
+      <p class="product-box__text">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et. dolore magna aliqua facilisis.
+      </p>
+      <a p class="product-box__link" href="#">
```

Comment:
jeżeli chciałbyś zrobić zadanie dodatkowe to ta strzałka powinna być przyciskiem, a nie linkiem

dodatkowo niepotrzebne "p" przed "class"